src/Components/HumanAtNasa/HumanAtNasa.tsx

```diff
@@ -0,0 +1,22 @@
+﻿import React from 'react';
+import './HumanAtNasa.scss';
+
+interface HumanAtNasaProps{
+    name : string;
+    jobTitle : string;
+    jobDescription: string;
+    imageUrl : string;
+}
+
+export function HumanAtNasa(props : HumanAtNasaProps){
+    
+    return(
+        <div className='humansComponent'>
+            <img className='humansImage' src={props.imageUrl}/>
+            <h2 className='humanNames'>{props.name}</h2>
+            <h3 className='jobTitle'>{props.jobTitle}</h3>
```

Comment:
Is this really a header? It feels like the 'name' is the header, and jobTitle is just a bit of information?

Reply:
I agree, changed it to <p>.